sympy/parsing/sympy_parser.py

```diff
@@ -848,7 +848,11 @@ def stringify_expr(s, local_dict, global_dict, transformations):
         tokens.append((toknum, tokval))
 
     for transform in transformations:
-        tokens = transform(tokens, local_dict, global_dict)
+        # TODO: yuck!
```

Comment:
That's why I suggested to make a second transformer that converts `Float(string)` into `Float(string, precision)`.

Reply:
That makes sense!  Sorry I didn't understand your original comment (I did this without really understanding any of this code in fact ;-)